plugins/aws/fix_plugin_aws/resource/base.py

```diff
@@ -556,13 +566,18 @@ def add_node(
         return node
 
     def add_edge(
-        self, from_node: BaseResource, edge_type: EdgeType = EdgeType.default, reverse: bool = False, **to_node: Any
+        self,
+        from_node: BaseResource,
+        edge_type: EdgeType = EdgeType.default,
+        reverse: bool = False,
+        permissions: Optional[List[AccessPermission]] = None,
```

Comment:
```suggestion
        reported: Optional[Json] = None,
```
Instead of adding raw permissions, I would create a dict as reported. 
The reported section itself can contain anything you want.